gradle.properties

```diff
@@ -59,14 +59,14 @@ SNAPSHOT_REPOSITORY_URL=https://oss.sonatype.org/content/repositories/snapshots/
 
 # Java Configuration -------------------------------------------------------------------------------
 
-compileJava.sourceCompatibility=1.7
-compileJava.targetCompatibility=1.7
+compileJava.sourceCompatibility=17
+compileJava.targetCompatibility=17
```

Comment:
so bumping the JDK to 17 as the minimum really should be updated in the readme and in the release.bat. not sure if that will cause issues for anyone.